src/Salt/SiteBundle/Entity/CommentRepository.php

```diff
@@ -76,4 +84,42 @@ public function findByTypeItem(array $id): array
     {
         return $this->findByItem($id['itemType'].':'.$id['itemId']);
     }
+    
+     /**
+     * @param string $itemType
+     * @param int $itemId
+     *
+     * @return Comment
+     */    
+    
+    public function exportComments($itemType,$itemId)
+    {
+        $query= $this->getEntityManager()
```

Comment:
Is there a special reason to create manually the query instead of just using Doctrine ORM?

Reply:
I want to fit everything in single query. So I preferred this way. If you have other better way which fulfills all requirements in single query, then would you mind suggesting that query.

Reply:
You can get all data you need with one single query with doctrine ORM, finding by `itemId`.